com.ibm.streamsx.plumbing/com.ibm.streamsx.plumbing.filters/namespace-info.spl

```diff
@@ -0,0 +1,17 @@
+/*
+*******************************************************************************
+* Copyright (C) 2016, International Business Machines Corporation. 
+* All Rights Reserved. *
+*******************************************************************************
+*/
+
+/**
+ * Enhanced filter functionality.
+ *
+ * TraceLevelFilter allows administrators to dynamically turn streams on and off
```

Comment:
Not sure the sentence is about the operator is needed here, at least maybe it shouldn't be a single paragraph, as what happens when additional operators are added.

Also, I feel the description here is a little strange, is that really the purpose of the operator, to allow admin control of an application? Seems it's more a diagnostic aid.